backend/API_DOC.md

```diff
@@ -157,3 +157,41 @@ Gets all the communities the user is a member of.
 ---
 
 ## Request
+
+
+---
+
+## Item
+
+### Update Found Status
+
+Updates the status of if an item is found or not. 
+
+**URL** : `http://api.good-grocer.click/item/updateStatus`
+
+**Method** : `POST`
+
+**Auth Required** : YES
+
+**Body Parameters** :
+
+```json
+{
+  "id": "[int, required id of the item]",
+  "found": "[bool, true if found, false is not]"
+}
+```
+
+**Success Response** : `200 OK`
+
+```json
+{
+  "item": {
+    "id":  "[int, id of the item]", 
+    "requested_by":  "[int, id of user that requested item]"
+    ... 
```

Comment:
Might be better to put all the fields in the response so they know what types to expect